clients/lsp-kotlin.el

```diff
@@ -146,7 +184,11 @@ to Kotlin."
                     (with-lsp-workspace workspace
                       (lsp--set-configuration (lsp-configuration-section "kotlin"))))
   :download-server-fn (lambda (_client callback error-callback _update?)
-                        (lsp-package-ensure 'kotlin-language-server callback error-callback))))
+                        (lsp-package-ensure 'kotlin-language-server callback error-callback))
+  :after-open-fn (lambda ()
+                   ;; set lens backends so they are available is lsp-lens-mode is activated
+                   ;; backend does not support lenses, and block our other ones from showing. When backend support lenses again, we can use cl-pushnew to add it to lsp-lens-backends instead of overwriting
+                   (setq-local lsp-lens-backends (list #'lsp-kotlin-lens-backend)))))
```

Comment:
can you wrap that in minor mode (similar to https://github.com/emacs-lsp/lsp-java/blob/update-sections/lsp-java.el#L1521 )